doc/api/fs.md

```diff
@@ -1583,6 +1583,29 @@ fs.open('<directory>', 'a+', (err, fd) => {
 });
 ```
 
+Some characters are reserved under Windows which documented by
```

Comment:
Can you write this as (or something similar):

Some characters are reserved on Windows. These characters are documented at Naming Files, Paths, and Namespaces.